arcor2_AREditor/Assets/BASE/Scripts/SceneManager.cs

```diff
@@ -421,7 +421,7 @@ public class SceneManager : Singleton<SceneManager> {
         /// </summary>
         /// <param name="sender">Who invoked event.</param>
         /// <param name="args">Robot joints data</param>
-        private async void RobotJointsUpdated(object sender, RobotJointsUpdatedEventArgs args) {
+        public async void RobotJointsUpdated(object sender, RobotJointsUpdatedEventArgs args) {
```

Comment:
Why is this public?

Reply:
While using the robot positioning menu, RobotJointsUpdated and RobotEefUpdated in SceneEditor are unsubscribed from the corresponding events, to allow the robot model to be moved independently of the real-world robot.